grpc_codegen/avs.proto

```diff
@@ -32,13 +32,42 @@ message CheckinResp {
   google.protobuf.Timestamp updated_at = 1;
 }
 
-message SyncTasksReq {
+message SyncMessagesReq {
   string id = 1;
   string address = 2;
-  string signature = 3;
+  bytes  signature = 3;
   int64  monotonic_clock = 4;
 }
 
+enum MessageOp {
+  Unset              = 0;
+  MonitorTaskTrigger = 1;
+  CancelTask         = 2;
+  DeleteTask         = 3;
+  CompletedTask      = 4;
+}
+
+message SyncMessagesResp {
```

Comment:
please skip these changes. we're split the protobuf into node vs user feature in next phase.

Reply:
Got it yes, I see this `SyncMessagesResp` widely used in the avs_pb.js now. 👌